buildkite/pipelines_test.go

```diff
@@ -262,6 +262,34 @@ func TestPipelinesService_AddWebhook(t *testing.T) {
 
 	_, err := client.Pipelines.AddWebhook("my-great-org", "my-great-pipeline-slug")
 	if err != nil {
-		t.Errorf("Pipelines.Delete returned error: %v", err)
```

Comment:
This is me fixing an error reporting from a previous test-case in a previous PR that I had made for adding AddWebhook support. Calling this out as it's unrelated to the rest of the PR.